packages/parany/parany.12.0.1/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: "Francois Berenger"
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/UnixJunkie/parany"
+bug-reports: "https://github.com/UnixJunkie/parany/issues"
+dev-repo: "git+https://github.com/UnixJunkie/parany.git"
+depends: [
+  "base-unix"
+  "cpu"
+  "dune" {>= "1.6.0"}
+  "ocaml" {>= "4.03.0"}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+# the test exe doesn't run in the CI env. of the opam-repository
+# probably because processes are limited to one core, I guess
+# ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
I'm curious, what was the error message you got?

Reply:
oh i see. Could the fix be merged first? https://github.com/UnixJunkie/parany/pull/48 It's always nice to have some tests

Reply:
in the dune file, can we use this "env var." instead of hard-cogin 16: `getconf _NPROCESSORS_ONLN` ?

Reply:
FYI: I launch the regression tests on my machine prior to releasing the software.
Does the CI env. will really allow me to run a parallel job?

Reply:
> in the dune file, can we use this "env var." instead of hard-cogin 16: `getconf _NPROCESSORS_ONLN` ?

`%{jobs}%`? Mh AFAIK only using an environment variable.

>  Does the CI env. will really allow me to run a parallel job?

yes, always. We do it ourselves.

Reply:
ok, I'll use %{jobs}, thanks.